packages/lit-dev-tests/src/playwright/util.ts

```diff
@@ -23,11 +23,42 @@ export async function waitForPlaygroundPreviewToLoad(page: Page) {
     });
     return true;
   }, iframe);
-  // Wait for the loading indicator to stop for screenshots.
-  await page.waitForSelector(
-    'playground-preview [part="preview-loading-indicator"][aria-hidden="true"]'
-  );
-  // There is a fade-out transition on the playground loading bar that makes
-  // screenshots flaky. Wait for the loading bar to have animated out.
-  await page.waitForTimeout(250);
+  // Hide the animated loading indicator.
+  await page.evaluate((el) => {
+    el.style.visibility = 'hidden';
+  }, await page.waitForSelector('playground-preview [part="preview-loading-indicator"]'));
+}
+
+interface MwcSnackbar extends HTMLElement {
+  timeoutMs: number;
+  open: boolean;
+}
+
+/**
+ * Prevent any <mwc-snackbar> elements on the page from automatically closing
+ * after they are opened, and disable their opacity/transform transitions.
```

Comment:
For wholesale removal of animations. Could that "reduce motion" setting be used?

Although that doesn't necessarily solve it. This is amazing.

Reply:
Doesn't look like MWC respects `prefers-reduced-motion` at all, except in this one place (the slider component, which we don't use):

https://github.com/material-components/material-components-web/search?q=prefers-reduced-motion

Plus reduced motion could still mean *some* motion, even if the component did chose to respect `prefers-reduced-motion`.